src/components/Card.svelte

```diff
@@ -11,7 +11,11 @@
 	let audio: HTMLAudioElement;
 
 	onMount(() => {
-		audio = new Audio(songDetails.previewAudio);
+		if (songDetails.previewAudio != 'null') {
+			audio = new Audio(songDetails.previewAudio);
+		} else {
+			audio = new Audio();
+		}
```

Comment:
Why not initialize it at line 11, then do a conditional check of
```
if (songDetails.previewAudio != 'null')
	audio = new Audio(songDetails.previewAudio);
```

Reply:
see comment below regarding != vs !==

Reply:
will update

Reply:
It is actually better to leave it like this because reassigning audio does not mutate the object you initialize it with. It only rebinds the pointer to point to a new obejct. By initializing and potentially creating multiple unused Audio objects, it may lead to decrease in performance.

Reply:
very valid explanation, sounds good to me